components/navigation.css

```
@@ -0,0 +1,23 @@
+.navigation {
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+  background-color: var(--color-white);
+  padding: 12px 29px;
+  border-radius: 10px;
+  box-shadow: 0 2px 11px var(--shadow-grey);
+  position: fixed;
+  bottom: 20px;
+  left: 20px;
+  right: 20px;
+}
+
+.navigation a {
+  text-decoration: none;
+  line-height: 0;
+}
+
+.navigation__icon-inactive {
```

To follow BEM, I would change to 
```suggestion
.navigation__icon-inactive {

.navigation__icon--inactive 

```